code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -182,73 +189,107 @@
+		if(MUTCOLORS in S.specflags)
+			if(S.fixed_mut_color)
+				species_color = S.fixed_mut_color
+			else
+				species_color = H.dna.features["mcolor"]
+			should_draw_greyscale = TRUE
+		else
+			species_color = ""
 
-	if(MUTCOLORS in S.specflags)
-		if(S.fixed_mut_color)
-			species_color = S.fixed_mut_color
+		if(!dropping_limb && H.dna.check_mutation(HULK))
+			mutation_color = "00aa00"
 		else
-			species_color = H.dna.features["mcolor"]
-		should_draw_greyscale = TRUE
-	else
-		species_color = ""
+			mutation_color = ""
+
+		dmg_overlay_type = S.damage_overlay_type
 
-	if(H.disabilities & HUSK)
-		species_id = "husk"
+	else if(animal_origin == MONKEY_BODYPART) //currently monkeys are the only non human mob to have damage overlays.
+		dmg_overlay_type = animal_origin
+
+	if(C.disabilities & HUSK)
+		species_id = "husk" //overrides species_id
+		dmg_overlay_type = "" //no damage overlay shown when husked
 		should_draw_gender = FALSE
 		should_draw_greyscale = FALSE
 
-	if(!dropping_limb && H.dna.check_mutation(HULK))
-		mutation_color = "00aa00"
-	else
-		mutation_color = ""
+	if(status == BODYPART_ROBOTIC)
+		dmg_overlay_type = "robotic"
 
 	if(dropping_limb)
 		no_update = 1 //when attached, the limb won't be affected by the appearance changes of its mob owner.
 
 //to update the bodypart's icon when not attached to a mob
 /obj/item/bodypart/proc/update_icon_dropped()
 	cut_overlays()
-	var/image/I = get_limb_icon(1)
-	if(I)
+	var/list/standing = get_limb_icon(1)
+	if(!standing.len)
+		icon_state = initial(icon_state)//no overlays found, we default back to initial icon.
+		return
+	for(var/image/I in standing)
 		I.pixel_x = px_x
 		I.pixel_y = px_y
-		add_overlay(I)
+	add_overlay(standing)
 
 //Gives you a proper icon appearance for the dismembered limb
 /obj/item/bodypart/proc/get_limb_icon(dropped)
-	var/image/I
+	icon_state = "" //to erase the default sprite, we're building the visual aspects of the bodypart through overlays alone.
+
+	var/list/standing = list()
+
+	var/image_dir
+	if(dropped)
+		image_dir = SOUTH
+		if(dmg_overlay_type)
+			if(brutestate)
+				standing	+= image("icon"='icons/mob/dam_mob.dmi', "icon_state"="[dmg_overlay_type]_[body_zone]_[brutestate]0", "layer"=-DAMAGE_LAYER, "dir"=image_dir)
+			if(burnstate)
+				standing	+= image("icon"='icons/mob/dam_mob.dmi', "icon_state"="[dmg_overlay_type]_[body_zone]_0[burnstate]", "layer"=-DAMAGE_LAYER, "dir"=image_dir)
+
+
+	if(animal_origin)
+		if(status == BODYPART_ORGANIC)
+			if(species_id == "husk")
+				standing += image("icon"='icons/mob/animal_parts.dmi', "icon_state"="[animal_origin]_husk_[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+			else
+				standing += image("icon"='icons/mob/animal_parts.dmi', "icon_state"="[animal_origin]_[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+		else
+			standing += image("icon"='icons/mob/augments.dmi', "icon_state"="[animal_origin]_[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
```

Comment:
this line does nothing since I gave up on adding robotic limb for monkeys, so there's no sprite for it.